assets/js/barChart.js

```diff
@@ -70,12 +74,27 @@ class BarChart {
 
         this.rects = bars.append('rect')
             .attr('y', function(d) { return yScale(d.name); })
-            .attr('height', yScale.rangeBand());
+            .attr('height', yScale.rangeBand())
+            // Add a tooltip to the bar.
+            .on("mouseover", function (d) { tooltip.style("display", "inline-block"); })
+            .on("mousemove", function (d) {
+                tooltip
+                    .style("left", d3.event.pageX - 50 + "px")
+                    .style("top", d3.event.pageY - 70 + "px")
+                    .style("display", "inline-block")
+                    .html((d.name.replace('<', '&lt;') + ' (' + d.val + ')'));
+            })
+            .on("mouseout", function (d) { tooltip.style("display", "none"); });
+
+        // Clear tooltip div created when inadvertently triggered during dropdown selection.
+        $( ".toolTip" ).remove();
```

Comment:
@jdaguil  This is needed at least on Windows 10.  Using the `Elements` view in the dev tools, with this code, we generate only a single toolTip div no matter how many times we select a different attribute for display:
![image](https://user-images.githubusercontent.com/11096678/29342632-a6ef52f8-81e0-11e7-8299-0fac87fb32b7.png)


When I comment out the code, a new div is created each time a different attribute is selected for display:
![image](https://user-images.githubusercontent.com/11096678/29342597-7a5f4180-81e0-11e7-92df-c49754459184.png)

I take it you see different behavior on your Mac?